chart/values.yaml

```diff
@@ -475,9 +476,11 @@ loki-stack:
     persistence:
       # -- Enable persistence storage for the logs
       enabled: true
-      # -- StorageClass for Loki's centralised log storage
-      # empty storageClass implies cluster default storageClass & `manual` creates a static hostpath PV
-      storageClassName: ""
+      # -- StorageClass for Loki's centralised log storage. Options:-
+      # <p> - `"openebs-hostpath"` (default) - Will provision openebs local-pv hostpath volume. Needs `Values.openebs-localpv-provisioner.enabled = true`. <br>
```

Comment:
Seems a bit clunky to have the same string defined here and in the comment above.
Default is already generated in the README so we can leave it out.

Reply:
They are both different values. So we need to put them at both places there. Not sure if there is any common place.

I had put the Default one there for the `Needs Values.openebs-localpv-provisioner.enabled = true` to make it clear from this value's document.